spec/controllers/idv/usps_controller_spec.rb

```diff
@@ -54,16 +53,16 @@
     end
 
     it 'logs an event when there is a timeout' do
-      allow_any_instance_of(ApplicationController).
-        to receive(:analytics).and_return(fake_analytics)
+      stub_analytics
```

Comment:
@stevegsa I cleaned up this spec to use our standard `stub_analytics` controller helper